packages/@guardian/source-foundations/src/typography/data.ts

```diff
@@ -101,6 +105,20 @@ export const fontWeights = {
 	bold: 700,
 } as const;
 
+export const fontWeightsAvailable = {
```

Comment:
I think something about this naming makes it harder to grasp whats going on here :) could this be `AvailableFontWeights` and the type called something else ie `AvailableFontWeightsType` or something?

Reply:
Also for my own interest were these defined previously or is this something we've added in this PR?

Reply:
good point, I agree about the name change 👍 

they (and the italic object below) were both defined previously as part of this object: https://github.com/guardian/source/pull/1525/files#diff-9f53feda13d0210df1793144b9787a8f820113e01c8b97eddaa9e391622d60eaL124

it was kind of a "two in one" scenario, where the presence of the font weight in that object meant that it was supported *and* then the italic boolean was defined a level below. 

so, this was an attempt to separate the two out and make it a bit clearer what the intent was. that said, I'm still not very happy with this solution - going to have a very quick play around with the shape of it to see if I can come up with a clearer way of expressing this

Reply:
couldn't think of a cleaner way to do this in the time box I gave myself — so i've renamed them as you suggested :)
Change commit here: [`d3bc35f` (#1525)](https://github.com/guardian/source/pull/1525/commits/d3bc35fb80b0d19c12b6b093992b63e2ab3f168c)

Reply:
Great, that works nicely now i think.